src/main/java/com/alkemy/ong/model/News.java

```diff
@@ -0,0 +1,51 @@
+package com.alkemy.ong.model;
+
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.hibernate.annotations.Where;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+
+import com.alkemy.ong.model.Category;
+@Getter
+@Setter
+@NoArgsConstructor
+
+@SQLDelete(sql = "UPDATE news SET softDelete = true WHERE id=?")
+@Where(clause = "softDelete = false")
+
+@Entity
+@Table(name="news")
+public class News {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotNull(message = "Name cannot be empty")
+    @Column(name = "name", length = 40, nullable = false)
```

Comment:
El _nullable = false_ que utilizas en todos los NOT NULL no esta mal, pero es redundante, ya que le estas poniendo tambien la annotation @NotNull